.github/scripts/muffet.sh

```diff
@@ -33,5 +33,8 @@ muffet http://localhost:1313 \
   --exclude "http[s]*://127\.0\.0*" \
   --exclude "http[s]*://.*rapidpro.io.*" \
   --exclude "http[s]*://.*africastalking.com*" \
+  --exclude "http[s]*://.*udemy.com.*" \
+  --exclude "http[s]*://.*notion.so.*" \
+  --exclude "http[s]*://.*medium.com.*" \
```

Comment:
Apologies if this is supposed to be obvious. Are medium links behind a paywall and that's why they don't resolve? If muffet not able to bypass the browser check?